build.cmd

```diff
@@ -389,6 +389,17 @@ REM ------------------ Report config -----------------------
 
 REM after this point, ARG variable should not be used, use only BUILD_* or TEST_*
 
+REM if the `PB_SKIPTESTS` variable is set to 'true' then no tests should be built or run, even if explicitly specified
+if /i "%PB_SKIPTESTS%" == "true" (
```

Comment:
Will this automatically thread through from Pipe build? I'm unsure if pipe build variables automatically become environment variables or not.